x-pack/plugins/security/server/authorization/authorization_service.tsx

```diff
@@ -178,7 +178,7 @@ export class AuthorizationService {
     http.registerOnPreResponse((request, preResponse, toolkit) => {
       if (preResponse.statusCode === 403 && canRedirectRequest(request)) {
         const next = `${http.basePath.get(request)}${request.url.pathname}${request.url.search}`;
-        const body = renderToStaticMarkup(
+        const body = renderToString(
```

Comment:
emotion does not support server side rendering via `renderToStaticMarkup()` so using `renderToString()` instead.